packages/react-admin-date-picker/src/DatePicker.sc.ts

```diff
@@ -1,7 +1,5 @@
 import zIndex from "@material-ui/core/styles/zIndex";
-import { mui } from "@vivid-planet/react-admin";
-
-const { css, styled } = mui;
+import { css, styled } from "@vivid-planet/react-admin";
```

Comment:
We should not export `styled` or `css` directly from react-admin, especially as we don't need this anymore. Nowadays this would be done using https://styled-components.com/docs/api#create-a-declarations-file

See for example https://github.com/vivid-planet/react-admin/blob/master/packages/react-admin-form/src/themeAugmentation.ts where we do that.

next release would be a good opportunity to completely switch to this technique and eliminate those two exports

(all of that that should be in a different PR)

Reply:
See #248